notebooks/official/experiments/get_started_with_comparing_pipeline_runs.ipynb

```diff
@@ -0,0 +1,1029 @@
+{
```

Comment:
<p>too sparse, explain what the cells do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>done</p>